OpenKh.Kh2/Motion.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Numerics;
+using Xe.BinaryMapper;
+
+namespace OpenKh.Kh2
+{
+    public class Motion
+    {
+        private class Header
+        {
+            [Data] public int Version { get; set; }
+            [Data] public int Unk04 { get; set; }
+            [Data] public int ByteCount { get; set; }
+            [Data] public int Unk0c { get; set; }
+        }
+
+        public class RawMotion
+        {
+            [Data] public int BoneCount { get; set; }
+            [Data] public int Unk14 { get; set; }
+            [Data] public int Unk18 { get; set; }
+            [Data] public int Unk1c { get; set; }
+            [Data] public int Unk20 { get; set; }
+            [Data] public int TotalFrameCount { get; set; }
+            [Data] public int Unk28 { get; set; }
+            [Data] public int Unk2c { get; set; }
+            [Data] public float Unk30 { get; set; }
```

Comment:
Unk30 ~ Unk4c seem to be a bounding box: min xyzw, and max xyzw

![2020-11-21_09h56_46](https://user-images.githubusercontent.com/5955540/99863086-f13de980-2bdf-11eb-8d53-51f6ed7eefcc.png)

Reply:
Good info, thanks! Meanwhile I found Unk2C usage too: I will push the changes as soon as I turn on my computer.